.rubocop.yml

```diff
@@ -204,6 +204,12 @@ RSpec/DescribeClass:
     - 'spec/features/**/*.rb'
     - 'modules/*/spec/features/**/*.rb'
 
+RSpec/IncludeExamples:
+  Exclude:
+    - 'spec/services/work_packages/shared/shared_examples_days.rb'
+    - 'spec/services/work_packages/shared/working_days_spec.rb'
+    - 'spec/services/work_packages/shared/all_days_spec.rb'
```

Comment:
I presume you're waiting for #19040 to be merged before removing?

Reply:
No, quite the inverse. I want to merge that one first because the tests were passing. But with your reviews I'll have to amend and force push it again anyway, so yes that's probably the right thing to do.